src/Message/AuthorizeRequest.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace MyOnlineStore\Omnipay\KlarnaCheckout\Message;
+
+use Klarna\Rest\Checkout\Order;
+
+/**
+ * Creates a Klarna Checkout order if it does not exist
+ */
+final class AuthorizeRequest extends AbstractRequest
+{
+    use ItemDataTrait;
+
+    /**
+     * @inheritDoc
+     */
+    public function getData()
+    {
+        $this->validate('terms_url');
```

Comment:
Optional: terms_url is not the only required parameter, the presence of more parameters could be asserted here